lib/api/database/chat_services.dart

```diff
@@ -39,28 +39,31 @@ class ChatServicesNotifier extends StateNotifier<List<ChatBubble>> {
   ChatBubble _parseChat(Chat chat) {
     return ChatBubble(
       margin: const EdgeInsets.only(top: 10),
-      child: Text(chat.message),
       alignment:
           user!.id == chat.senderid ? Alignment.topRight : Alignment.topLeft,
       shadowColor: Colors.transparent,
       backGroundColor: user!.id != chat.senderid
           ? Colors.grey
           : NoSignalTheme.lightBlueShade,
       clipper: ChatBubbleClipper1(
-          type: user!.id == chat.senderid
-              ? BubbleType.sendBubble
-              : BubbleType.receiverBubble),
+        type: user!.id == chat.senderid
+            ? BubbleType.sendBubble
+            : BubbleType.receiverBubble,
+      ),
+      child: Text(chat.message),
     );
   }
 
   /// Constructor
   ChatServicesNotifier(
       {required this.client, this.user, required this.collectionId})
       : super([]) {
-    database = Database(client);
+    database = Databases(client, databaseId: 'YOUR DATABASE ID');
```

Comment:
should be `ApiInfo.databaseId`

Reply:
I'll change this 💯 tomorrow. It's a bit already night-time here.